namer/consul/src/main/scala/io/buoyant/namer/consul/LookupCache.scala

```diff
@@ -28,7 +28,6 @@ private[consul] class LookupCache(
 
   private[this] val localDcMoniker = ".local"
 
-
```

Comment:
I don't know how this extra whitespace got here in the first place or why scalariform suddenly decided it needed to fix it right now.